app/components/trainees/confirmation/degrees/view.rb

```diff
@@ -2,11 +2,12 @@ module Trainees
   module Confirmation
     module Degrees
       class View < GovukComponent::Base
-        attr_accessor :degrees, :trainee
+        attr_accessor :degrees, :trainee, :show_add_another_degree_button
 
-        def initialize(trainee:)
+        def initialize(trainee:, show_add_another_degree_button: true)
```

Comment:
It looks like the production use of this code is having `show_add_another_degree_button` set to `false` and only true for `Trainees::Confirmation::Degrees::ViewPreview` and another spec file. I suggest having the default value as `false` and switching it on for preview mode and test mode.